plugins/generator-1.15.2/forge-1.15.2/templates/tab.java.ftl

```diff
@@ -27,30 +27,31 @@
  # exception.
 -->
 
-<#-- @formatter:off -->
-<#include "mcitems.ftl">
-
-package ${package}.itemgroup;
-
-@${JavaModName}Elements.ModElement.Tag
-public class ${name}ItemGroup extends ${JavaModName}Elements.ModElement{
-
-	public ${name}ItemGroup (${JavaModName}Elements instance) {
-		super(instance, ${data.getModElement().getSortID()});
-	}
-
-	@Override public void initElements() {
-		tab = new ItemGroup("tab${registryname}") {
-			@OnlyIn(Dist.CLIENT) @Override public ItemStack createIcon() {
-				return ${mappedMCItemToItemStackCode(data.icon, 1)};
-			}
-			@OnlyIn(Dist.CLIENT) public boolean hasSearchBar() {
-				return ${data.showSearch};
-			}
-		}<#if data.showSearch>.setBackgroundImageName("item_search.png")</#if>;
-	}
-
-	public static ItemGroup tab;
-
-}
+<#-- @formatter:off -->
```

Comment:
Could you check why this whole section is marked as changed, maybe you changed white spaces type?